configuration/shared_subscriptions.md

```diff
@@ -22,3 +22,23 @@ shared_subscription_policy = prefer_local
 
 When a messages is being delivered to subscribers of a shared subscription, the message will be delivered to an online subscriber if possible, otherwise the message will be delivered to an offline subscriber.
 
+## Examples
+
+**Subscriptions**
+*Note: When subscribing to a shared topic, make sure to escape the `$`*
+```bash
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t $share/group/topicname
+mosquitto_sub -h mqtt.example.io -p 1883 -q 2 -t $share/group/topicname/#
```

Comment:
I'm not sure that we should show an example which is actually incorrect - from experience people copy the first thing they see and do not read on for the correct one.  I would leave out the unescaped example completely.